docs/pages/reference/access-controls/access-monitoring-rules.mdx

```diff
@@ -0,0 +1,271 @@
+---
+title: Access Monitoring Rule Reference
+description: An explanation and overview of Access Monitoring Rules.
+---
+
+Access Monitoring Rules allow administrators to monitor Access Requests and apply
+notification routing rules or automatic review rules based on specific conditions.
+
+An Access Monitoring Rule can configure both a notification routing rule and an
+automatic review rule, or it can contain only one.
+
+## YAML specification
+
+```yaml
+kind: access_monitoring_rule
+version: v1
+metadata:
+  name: example_rule
+spec:
+  # subjects specifies the kinds of subjects to monitor.
+  # Possible values: "access_request"
+  subjects:
+  - access_request
+
+  # condition specifies the conditions that should be met to apply the access
+  # monitoring rule. The condition accepts a predicate expression which must
+  # evaluate to a boolean value.
+  #
+  # This condition would be satisfied by an access request if the `access` role
+  # is requested, and if the requesting user has the `team: dev` user trait.
+  condition: |-
+    contains_all(set("access"), access_request.spec.roles) &&
```

Comment:
`contains_all` with a set containing a single element is basically the same as `contains_any`, isn't it? Should we come up with a better example that illustrates `contains_all` or just change these both to `contains`?